### DISCUSSION TITLE
Remember previous window size and state

### DISCUSSION THREAD

**Discussion by focusright** [phcode-dev/phoenix#1794] [Ideas]
Upon opening the program, it does not remember the size and state of the program window upon previous close; the window state being maximized or otherwise, and if not then the window size. This feature would be nice to have.

**Comment by abose** [comment]
Thanks for reporting the issue @focusright . We had this feature when we released it, but looks like its borken after a tauri upgrade.

Created an issue to track this: https://github.com/phcode-dev/phoenix/issues/1796
